code/modules/mob/mob_defines.dm

```diff
@@ -79,6 +79,7 @@
 	var/obj/item/r_hand = null//Living
 	var/obj/item/weapon/storage/s_active = null//Carbon
 
+	var/seer = 0 //for cult//Carbon, probably Human
```

Comment:
One of the many lines where I said "why does this exist?" and answered "I dont know, better add it anyway".